### PR TITLE
Don't include linkings in developer database dump

### PR DESCRIPTION
**Pull request by jonatanklosko**
Today while running `RAILS_ENV=production bin/rake db:dump:development` manually on production, I run into the following error:
```
Creating temporary database 'wca_development_db_dump'...done in 1.57s
Populating sanitized tables in 'wca_development_db_dump'...rake aborted!
ActiveRecord::StatementInvalid: Mysql2::Error: The table 'linkings' is full: INSERT INTO wca_development_db_dump.linkings (wca_id, wca_ids) SELECT linkings.wca_id, linkings.wca_ids FROM linkings
```

Given that the `linkings` table is not an important one, and can be computed locally if needed, excluding it from the developer dump seems just fine. This solves the problem.

### CODE REVIEW

File: WcaOnRails/lib/database_dumper.rb [hunk 1]
```diff
@@ -570,15 +570,9 @@ def self.actions_to_column_sanitizers(columns_by_action)
...
-      where_clause: "",
-      column_sanitizers: actions_to_column_sanitizers(
-        copy: %w(
-          wca_id
-          wca_ids
-        ),
-      ),
-    }.freeze,
+    # The `linkings` table is a big one and not really important one,
+    # so we avoid copying all the rows. It has even led to mysql full table error.
```
Comment (jfly) [inline WcaOnRails/lib/database_dumper.rb]: uber nit: This comes across a bit awkwardly. I'd rephrase it like this:

> We have seen MySQL full table errors when trying to copy the entire `linkings` table. Fortunately, it is a big and not really important table, so we can simply skip all its rows.

Comment (jonatanklosko) [reply WcaOnRails/lib/database_dumper.rb]: Sounds way better, thanks! There is a funny part though :smiley: 
> Fortunately, it is a big [...] table

Comment (jfly) [reply WcaOnRails/lib/database_dumper.rb]: Oh whoops! That's not fortunate =) How about simply: "Fortunately, it is a not really important table..."

Comment (jonatanklosko) [reply WcaOnRails/lib/database_dumper.rb]: Yep, that's what I did!
